Proton/Sniffs/Commenting/InlineCommentSniff.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Proton\Sniffs\Commenting;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+class InlineCommentSniff implements Sniff
+{
+    /**
+     * @inheritDoc
+     */
+    public function register(): array
+    {
+        return [T_COMMENT];
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function process(File $phpcsFile, $stackPtr): void
+    {
+        $tokens = $phpcsFile->getTokens();
+
```

Comment:
```suggestion

        if (
            ($tokens[$stackPtr]['content'][0] === '#' && $tokens[$stackPtr]['length'] <= 1)
            || ($tokens[$stackPtr]['content'][0] !== '#' && $tokens[$stackPtr]['length'] <= 2)
        ) {
            return;
        }
```
I think it would be nice to add something like that, to avoid having the message when we have an empty comment (like `//`)

Reply:
I added a check for this and this sniff does not complain about those kind of comments anymore ("empty line comments"). Disallowing those comments is part of another rule (`SlevomatCodingStandard.Commenting.EmptyComment`) - already included in the `ruleset.xml`, too